requirements-dev.txt

```diff
@@ -6,3 +6,4 @@ transifex-client==0.12.1
 edx-i18n-tools==0.4.5
 pycodestyle==2.4.0
 pylint==0.28.0
+django-pyfs==2.0
```

Comment:
Not sure why this was added.

Reply:
@xitij2000 I  had to install `django-pyfs` manually to be able to run the test script. I will remove this since it's not part of the task.